broskichat-app/src/Components/Contact/Contact.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+
+
+class Contact extends React.Component {
+  render() {
+    return (
+      <div>
+
+        <div class="tile is-ancestor">
```

Comment:
@mattkim8 i think all of the `class="..."` should be changed to `className="..."` to follow React conventions.